derive/src/lib.rs

```diff
@@ -145,7 +145,7 @@ fn impl_eth_dispatch(
 
 						#result_instance
 
-						::pwasm_ethereum::ext::call(&self.address, self.value.clone().unwrap_or(::bigint::U256::zero()), &payload, &mut result[..])
+						::pwasm_ethereum::ext::call(self.gas.unwrap_or(0u64), &self.address, self.value.clone().unwrap_or(::bigint::U256::zero()), &payload, &mut result[..])
```

Comment:
i think better is `unwrap_or(::pwasm_ethereum::ext::gas_left())` so that default call won't always fail :)

Reply:
ah, I though that zero is a "fallback" value. So if it's not a case it means user who don't specify gas on in the call have to pay extra gas on ::pwasm_ethereum::ext::gas_left() call?

Reply:
it's miserable fee for this extern